satpy/etc/areas.yaml

```diff
@@ -1,3 +1,212 @@
+# This file contains a set of pre-defined areas to be used for resampling purposes.
+
+# ----------------------------------------------------------------------------------------------------------------------
```

Comment:
Thoughts on an 80 character limit for this file? There are ways in YAML to make multiline strings if we need to (for the area "description" if those get long.

Reply:
I have no opinion on this, so as you wish.

I guess we could go with:
```yaml
  description: >-
    MSG SEVIRI Full Earth Scanning service
    area definition with 3 km resolution
```

Reply:
Yeah I just noticed that it was hard to read all the section headers because of how long they were. Could have been my browser window width or GitHub imposing a character/column limit.

Reply:
Done! Noticed that simply splitting the lines without the `>-` works the same (folding with a space, no end newline).